pkg/api/folder_test.go

```diff
@@ -572,10 +572,14 @@ func TestUpdateFolderLegacyAndUnifiedStorage(t *testing.T) {
 	mux := http.NewServeMux()
 
 	mux.HandleFunc("GET /apis/folder.grafana.app/v0alpha1/namespaces/default/folders/ady4yobv315a8e", func(w http.ResponseWriter, req *http.Request) {
+		w.Header().Add("Content-Type", "application/json")
```

Comment:
This took me a while to track down!

client-go seems more strict about the Content-Type header in v1.32 -- previously this was actually sending "plan/text" when no header is specified and in the new 0.32.0, that would fail!